README.md

```diff
@@ -29,6 +29,11 @@ It is tiny (2kB, including dependencies).
 >[“It's cool that you are inventing a better Flux by not doing Flux at all.”](https://twitter.com/andrestaltz/status/616271392930201604)  
 >André Staltz, creator of Cycle
 
+### Before Proceeding Further
+
+>**Also read why you might not be needing Redux:
```

Comment:
If you intend to break the line here, add two spaces after the line end. (Yes, Markdown is weird!)